samples/meetings-attendance-report/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "<<Microsoft-App-Id>>",
```

Comment:
<<Microsoft-App-Id>> => Rename the manifest GUID name and update it accordingly in Readme file.